src/Facades/Stache.php

```diff
@@ -11,6 +11,7 @@
  * @method static self registerStores($stores)
  * @method static mixed stores()
  * @method static mixed store($key)
+ * @method static \Illuminate\Contracts\Cache\Store cache()
```

Comment:
This should be cacheStore() not cache() ?

Reply:
Right! I named it `cache` first but then saw that Glide uses `cacheStore` so switched to that